UnityPy/helpers/PackedBitVector.py

```diff
@@ -75,7 +75,14 @@ def unpack_floats(
 
     # read as int and cast up to double to prevent loss of precision
     quantized_f64 = unpack_ints(packed, start, count)
-    scale = packed.m_Range / ((1 << packed.m_BitSize) - 1)
+
+    # avoid zero division
+    scale_denominator = (1 << packed.m_BitSize) - 1
+    if scale_denominator == 0:
```

Comment:
In the original C++ code we have

```cpp
uint32_t x;
float scale;

x &= (1 << m_BitSize) - 1;
data[i] = (x / (scale * ((1 << (m_BitSize)) - 1))) + m_Start;
```

Assuming that m_BitSize is 0, then x is 0 as well.
With this we get

```cpp
data[i] = (0 / (scale * 0)) + m_Start;
```

Based on the IEEE 754 rules:

finite * 0.0 = 0.0 → 0 / 0.0 = NaN
±∞ * 0.0 = NaN → 0 / NaN = NaN
NaN * 0.0 = NaN → 0 / NaN = NaN

so we the result will always be NaN.

Which means that we can effectively return early with an array of NaN.